src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,88 @@
+package duke.parser;
+
+import duke.command.Command;
+import duke.command.CommandType;
+import duke.exception.DukeException;
+import duke.ui.Ui;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+/**
+ * Parser that parses raw user input into digestible output
+ */
+public class Parser {
+    /**
+     * Returns the description excluding the command type in the command string
+     *
+     * @param commandType   of the commandString
```

Comment:
I think there seems to be extra white-space here.

Reply:
Yes you are right! I will fix it 🔨

Reply:
Actually, I realise the formatting is automatically done by intellij to align the comment with the next one, for readability purpose. I don't think it will affect the docs generated, so I will leave it there.